### ISSUE TITLE
[1.0.0-alpha2] Timepicker in modal shows overlay in top of everything

### ISSUE THREAD

**Issue by Magiczne** [Dogfalo/materialize#5443]
## Expected Behavior
It should open timepicker above modal

## Current Behavior
Timepicker opens, but at the top of everything there is an overlay.

## Steps to Reproduce (for bugs)

CodePen: [https://codepen.io/anon/pen/oomQzy](https://codepen.io/anon/pen/oomQzy)

## Your Environment
Version used: 1.0.0 alpha 2
Browser Name and version: Chrome 62.0.3202.94
Operating System and version (desktop or mobile): Windows 10

**Comment by huww98** [comment]
@Magiczne  You can use the `container` option

````javascript
var time_instance = new M.Timepicker(time_elem, {container: "body"})
````

https://codepen.io/anon/pen/EbzrdB

**Comment by Magiczne** [comment]
@huww98 Yeah, i know, but in the previous versions of materialize it was perfectly fine

**Comment by Dogfalo** [comment]
Fixed in fafeb0a
